src/modules/extra/m_ssl_gnutls.cpp

```diff
@@ -321,6 +321,10 @@ class ModuleSSLGnuTLS : public Module
 
 		ServerInstance->Modules->AddService(iohook);
 		ServerInstance->Modules->AddService(starttls);
+
+		if (starttls.enabled && !ServerInstance->Modules->Find("m_cap.so"))
+			ServerInstance->Logs->Log("m_ssl_gnutls", DEFAULT, "WARNING: m_cap.so is not loaded. The tls capability will not be available to clients without this!");
```

Comment:
I think adding this would create more confusion than help anyone. The wording "The tls capability will not be available to clients without this" can be misunderstood as "no TLS available until you load m_cap". STARTTLS is enabled by default meaning anyone who loads m_ssl_gnutls but doesn't load m_cap would get this warning. The `tls` cap is mostly ignored anyway so users lose next to nothing or nothing at all, so please remove the check from here.